sources/dialect/README.MD

```diff
@@ -0,0 +1,19 @@
+# Dialect Support
```

Comment:
README.md

No need to introduce a .MD extension

Reply:
That's a typo on my part, I'll fix it shortly.